docs/sources/flow/reference/components/discovery.hetzner.md

```diff
@@ -29,20 +29,23 @@ discovery.hetzner "LABEL" {
 
 The following arguments are supported:
 
-Name | Type | Description | Default | Required
----- | ---- | ----------- | ------- | --------
-`role` | `string` | Hetzner role of entities that should be discovered. | | yes
-`port` | `int` | The port to scrape metrics from. | `80` | no
-`refresh_interval` | `duration` | The time after which the servers are refreshed. | `"60s"` | no
-`bearer_token` | `secret` | Bearer token to authenticate with. | | no
-`bearer_token_file` | `string` | File containing a bearer token to authenticate with. | | no
-`proxy_url` | `string` | HTTP proxy to proxy requests through. | | no
-`follow_redirects` | `bool` | Whether redirects returned by the server should be followed. | `true` | no
-`enable_http2` | `bool` | Whether HTTP2 is supported for requests. | `true` | no
+Name                     | Type                | Description                                                   | Default | Required
+------------------------ | ------------------- | ------------------------------------------------------------- | ------- | --------
+`role`                   | `string`            | Hetzner role of entities that should be discovered.           |         | yes
+`port`                   | `int`               | The port to scrape metrics from.                              | `80`    | no
+`refresh_interval`       | `duration`          | The time after which the servers are refreshed.               | `"60s"` | no
+`bearer_token_file`      | `string`            | File containing a bearer token to authenticate with.          |         | no
+`bearer_token`           | `secret`            | Bearer token to authenticate with.                            |         | no
+`enable_http2`           | `bool`              | Whether HTTP2 is supported for requests.                      | `true`  | no
+`follow_redirects`       | `bool`              | Whether redirects returned by the server should be followed.  | `true`  | no
+`proxy_url`              | `string`            | HTTP proxy to send requests through.                          |         | no
+`no_proxy`               | `string`            | Comma-separated list to exclude from proxying.                |         | no
+`proxy_from_environment` | `bool`              | Use proxy URL indicated by environment variables.             | `false` | no
+`proxy_connect_header`   | `map(list(secret))` | Specifies headers to send to proxies during CONNECT requests. |         | no
 
 `role` must be one of `robot` or `hcloud`.
 
- You can provide one of the following arguments for authentication:
+ At most one of the following can be provided:
```

Comment:
```suggestion
 At most, one of the following can be provided:
```

I only tagged this one instance with the suggestion for adding a comma. It's probably easier to search/replace in the branch vs trying to find every instance of this and post a suggested change to add a comma.